examples/game/breakout.rs

```diff
@@ -185,7 +185,7 @@ fn paddle_movement_system(
     keyboard_input: Res<Input<KeyCode>>,
     mut query: Query<(&Paddle, &mut Transform)>,
 ) {
-    if let Ok((paddle, mut transform)) = query.single_mut() {
+    if let Ok((paddle, mut transform)) = query.try_single_mut() {
```

Comment:
I think this will be clearer with just a standard single_mut.

Reply:
Good call, I removed all the `try_` from the breakout example.